test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkParseTests.cs

```diff
@@ -248,14 +235,19 @@ public void NuGetFramework_ParseToShortName(string expected, string fullName)
         }
 
         [Theory]
+        // Net5.0 ERA
+        [InlineData("net5.0", ".NETFramework,Version=v5.0")]
+        [InlineData("net10.1.2.3", ".NETFramework,Version=v10.1.2.3")]
+        [InlineData("netcoreapp5.0", ".NETFramework,Version=v5.0")]
+
+        // Pre-Net5.0 ERA
```

Comment:
I don't see anything validating `net10` is `.NETFramework,Version=v1.0`. Also, I'd suggest a `net10.0`, I don't think the `net10.1.2.3` test is enough.

Reply:
next commit will cover both of those.